app/assess/templates/macros/banner_summary.html

```diff
@@ -13,7 +13,15 @@ <h3 class="govuk-body-l fsd-banner-content fsd-banner-collapse-padding">Project
                         name: {{ project_name }}</h3>
                     <h3 class="govuk-body-l fsd-banner-content">Total funding requested:
                         &pound;{{ "{:,.2f}".format(funding_amount_requested | float) }}</h3>
-                    <h3 class="govuk-body-l fsd-banner-content fsd-banner-collapse-padding">{{ workflow_status | status_to_human }}</h3>
+                    <h3 class="govuk-body-l fsd-banner-content fsd-banner-collapse-padding">
+                        {% if flag == None  %}
+                            {{ workflow_status | status_to_human }}
+                        {% else %}
+                            {{ flag.flag_type.name | capitalize }}
+                        {% endif %}
```

Comment:
I'd probably flip this and go with:
```suggestion
                        {% if flag %}
                            {{ flag.flag_type.name | capitalize }}
                        {% else %}
                             {{ workflow_status | status_to_human }}
                        {% endif %}
```

Seems more intuitive to me, but to each their own (not a big deal)

Reply:
You're right!